projects/common/src/error_collection.rb

```diff
@@ -25,14 +18,12 @@ def has_errors?
       !empty?
     end
 
-    foobara_delegate :empty?, :partition, :size, :clear, to: :error_array
-
     def errors
-      error_array
+      self
     end
```

Comment:
Feels like this method should be deprecated since the error collection is now also an array of errors. I don't have a pattern established yet for deprecated methods but we could just try `$stderr.puts "DEPRECATED: Do not call ErrorCollection#errors instead just use the collection directly.`